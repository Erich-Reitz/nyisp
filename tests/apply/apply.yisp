(set numbers '(4 1 7 3 2))

(apply + numbers)

(set lengths '(5 10 15))

(apply * lengths)
