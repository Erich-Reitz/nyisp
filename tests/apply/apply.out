nil
17
nil
750
